### DISCUSSION TITLE
problem with drawing outside “box”

### DISCUSSION THREAD

**Discussion by Coffeemoth** [keiwando/toonsquid#494] [General]
Hey it’s me once more and I need help again for this mysterious box issue. I can’t draw outside it and it moves with my drawing please help
![IMG_0278](https://github.com/keiwando/toonsquid/assets/153572884/765caf6c-aaa4-4fd8-93b7-ee0b708593b4)

**Comment by keiwando** [comment]
https://toonsquid.com/handbook/guides/faq/#pixel-layer-limit-border

**Reply by Coffeemoth** [reply]
Thank you, I tried reading it but I can’t find anything that fixes my problem. For example, how do I reset my pixel layer boundaries? And Is there a way to turn off the boundaries?

**Reply by keiwando** [reply]
There is no way to turn off the limits. Pixel layers in ToonSquid have a limited resolution, outside of which you can't draw. You can add a camera layer and use the crop tool to increase the resolution of your project outside of the camera region if it's not already at the 4k resolution limit and if you want a bit more space to draw outside of the region visible in the final export with pixel layers.

You can move the layer again using either the [transform inputs in the inspector](https://toonsquid.com/handbook/transform/transform_properties/#transform) or with the transform tool after disabling the [Edit pixels](https://toonsquid.com/handbook/transform/transform_tool/#tool-options) option.
